docs/resources/block_storage_volume.md

```diff
@@ -0,0 +1,57 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "exoscale_block_storage_volume Resource - terraform-provider-exoscale"
+subcategory: ""
+description: |-
+  Manage Exoscale Block Storage https://community.exoscale.com/documentation/block-storage/ Volume.
+  Block Storage offers persistent externally attached volumes for your workloads.
+---
+
+# exoscale_block_storage_volume (Resource)
+
+Manage [Exoscale Block Storage](https://community.exoscale.com/documentation/block-storage/) Volume.
+
+Block Storage offers persistent externally attached volumes for your workloads.
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) ❗ Volume name.
```

Comment:
Just to notice here, there is an upcoming call `UpdateBlockStorageVolume ` to update the name/labels of a block storage volume, so in a near future this will be not immutable.
Shipping it immutable like that now, and update it after should not be a blocker or breaking change, no?

Reply:
It will not break anything when we remove `RequreReplace`, so no problem here.